.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -78,7 +78,10 @@ jobs:
       - name: Notify failure in Slack
         uses: slackapi/slack-github-action@37ebaef184d7626c5f204ab8d3baff4262dd30f0 # v1.27.0
         with:
-          channel-id: int-cp-operator
-          slack-message: ":x: E2E ondemand tests failed on ${{ env.branch }} branch (${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
+          payload: |
+            {
+              "message": "ONDEMAND\ntests ${{needs.run-in-k8s.result == 'success' && 'passed :green_heavy_check_mark:' || 'failed :red_circle:'}} on kubernetes :kubernetes:\ntests ${{needs.run-in-ocp.result == 'success' && 'passed :green_heavy_check_mark:' || 'failed :red_circle:'}} on openshift :openshift:\n using ${{ env.branch }} branch (${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})",
```

Comment:
`run-in-k8s-` and `run-in-ocp` jobs don't exist here. we can only get the result of `run-matrix`

Reply:
copy paste error.... thanks